doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+
+[Top](#top)
+
+###<a name="section5.1"></a>Zookeeper
+####<a name=”section5.1.1></a>Configuration
+`ZooKeeper` is a centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services. The connections to `Zookeeper` use the port 2181 and provides the link between itself and brokers. Using the API we can create and ask for information about topics, produce some messages and consume, but the most of the actions are done automatically by `Cygnus`. The only action required for this architecture is to consume, that will carry out Sinfonier.
+
+`Zookeeper`is configured through the following parameters:
+
+| Parameter | Mandatory | Value | Comments |
+|---|---|---|---|
+| dataDir | yes | /tmp/zookeeper | Directory where the snapshot is stored (Default value) |
+| clientPort | yes | 2181 | Default port for Zookeeper |
+| maxClientCnxns | yes | 0 | Default value |
+
+`Zookeeper`configuration must be stored in a file called `zookeeper.properties`
+
+
+####<a name=”section5.1.2></a>Running
+Run `Zookeeper`is possible with a command:
+```
+bin/zookeeper-server-start.sh config/zookeeper.properties
+```
+An own terminal is required for this service. Another way is using `nohup` at the start and `&`at the end for running the server in background (Use restricted when brokers are running properly and if you know how `nohup` works)
+```
+nohup bin/zookeeper-server-start.sh config/zookeeper.properties &
+```
+
+[Top](#top)
+
+###<a name="section5.2"></a>Brokers
+####<a name=”section5.2.1></a>Configuration
+`Brokers` are used to distribute the information stored in `Kafka` and you can use one or several of them. This example use three `Brokers` and its configuration is described in this section. `Brokers` are implemented into `Zookeeper`, each one with its own port, i.e, a `Broker` with port 9092, another broker with port 9093, etc.
+A single `Broker` is configured through the following parameters:
+
+| Parameter | Mandatory | Value | Comments |
+|---|---|---|---|
+| broker.id | yes | 1 | An unique integer that define a broker |
+| port | yes | 9092 |  Port assigned to broker 1 |
+| host.name | yes | 0.0.0.0 |  For local connections must be `localhost`. For remote connections `0.0.0.0` allow them |
+| zookeeper.connect | yes | localhost:2181 | Usually zookeeper is running on a local machine. In other case, remote ip access is required with port 2181 |
+| log.dirs | yes | /tmp/kafka-logs-1 | kafka-logs-x, being x the broker.id |
+| advertised.host.name | no | your_remote_ip |  For remote connections of a known ip that allow it to access to the `broker` |
+| advertised.port | no | 9092 | For use a remote port different from the port broker set previously |
+
+A “multibroker” configuration can be configured creating different files, one by `Broker`, and setting same parameters like first, but changing `broker.id`, `port` and `logs.dir`.
```

Comment:
creating --> by creating
one by broker --> one per broker
like first --> like the first one